api/v1alpha1/ext_proc_types.go

```diff
@@ -70,4 +70,16 @@ type ExtProc struct {
 	//
 	// +optional
 	ProcessingMode *ExtProcProcessingMode `json:"processingMode,omitempty"`
+
+	// MetadataOptions defines the metadata that should be forwarded to or received from the external processor.
+	//
+	// +optional
+	Metadata *MetadataOptions `json:"metadata,omitempty"`
+}
+
+type MetadataOptions struct {
+	// ForwardingNamespaces is a list of namespaces from which metadata should be forwarded to the external processor.
+	ForwardingNamespaces []string `json:"forwardingNamespaces,omitempty"`
```

Comment:
Will this be translated to typed or untyped namespaces? if we want to support both options in the future, we maybe want to make the distinction here already, like here: https://github.com/envoyproxy/gateway/pull/3247/files#diff-2d475e566eff6abe27d7ed2892b3dd252980f2cfccc04bb859a14878c8ca71b1R63

Reply:
can you resubmit that one instead of this?

Reply:
it's stale, let's just go ahead with your PR.